scripts/utils/v4X_compatibility_test.sh

```diff
@@ -39,3 +40,9 @@ do
 done
 
 ./bootstrap; ./configure --enable-unit; make -j8 ; make check -j8
+
+if [ $? != 0 ];then
```

Comment:
Unneeded if `set -e` is present (same as `set -xe`, only with `set -xe` we're also telling bash to echo to stderr all the commands we're running. Which is helpful to know where the script failed if it does fail)